sechub-webui/src/test/java/com/mercedesbenz/sechub/webui/security/Base64EncodedClientIdAndSecretOAuth2AccessTokenClientTest.java

```diff
@@ -0,0 +1,187 @@
+    private static final Base64EncodedClientIdAndSecretOAuth2AccessTokenClient client = new Base64EncodedClientIdAndSecretOAuth2AccessTokenClient(restTemplate);
+    private static final ClientRegistration clientRegistration = ClientRegistration.withRegistrationId(Constants.REGISTRATION_ID)
+            .clientId(Constants.CLIENT_ID)
+            .clientSecret(Constants.CLIENT_SECRET)
+            .authorizationGrantType(AuthorizationGrantType.AUTHORIZATION_CODE)
+            .redirectUri(Constants.REDIRECT_URI)
+            .tokenUri(Constants.TOKEN_URI)
+            .authorizationUri(Constants.AUTHORIZATION_URI)
+            .build();
+    private static final OAuth2AuthorizationRequest authorizationRequest = OAuth2AuthorizationRequest.authorizationCode()
+            .authorizationUri(Constants.AUTHORIZATION_URI)
+            .clientId(Constants.CLIENT_ID)
+            .redirectUri(Constants.REDIRECT_URI)
+            .scopes(Set.of(Constants.OPENID))
+            .state(Constants.STATE)
+            .build();
+    private static final OAuth2AuthorizationResponse authorizationResponse = OAuth2AuthorizationResponse.success(Constants.CODE)
+            .redirectUri(Constants.REDIRECT_URI)
+            .state(Constants.STATE)
+            .build();
+    // @formatter:on
+    private static final OAuth2AuthorizationExchange authorizationExchange = new OAuth2AuthorizationExchange(authorizationRequest, authorizationResponse);
+    private static final OAuth2AuthorizationCodeGrantRequest authorizationCodeGrantRequest = new OAuth2AuthorizationCodeGrantRequest(clientRegistration,
+            authorizationExchange);
+    private static final String jwtResponseJson;
+
+    static {
+        try {
+            jwtResponseJson = Files.readString(Paths.get("src/test/resources/jwt-response.json"));
+        } catch (IOException e) {
+            throw new TestAbortedException("Failed to prepare test", e);
+        }
+    }
+
+    @BeforeEach
+    void beforeEach() {
+        mockServer.reset();
+    }
+
+    @Test
+    void get_token_response_executes_correctly_formatted_http_request() {
+        // prepare
+        String tokenUri = clientRegistration.getProviderDetails().getTokenUri();
+        String authorizationHeaderValue = getBasicAuthHeaderValue(clientRegistration.getClientId(), clientRegistration.getClientSecret());
+        // @formatter:off
+        mockServer.expect(requestTo(tokenUri))
+                .andExpect(method(HttpMethod.POST))
+                .andExpect(header(HttpHeaders.CONTENT_TYPE, Constants.APPLICATION_FORM_URLENCODED_VALUE))
+                .andExpect(header(HttpHeaders.AUTHORIZATION, authorizationHeaderValue))
+                .andExpect(content().formData(getMultiValueMap(authorizationCodeGrantRequest)))
+                .andRespond(withSuccess(jwtResponseJson, MediaType.APPLICATION_JSON));
+        // @formatter:on
+
+        // execute
+        client.getTokenResponse(authorizationCodeGrantRequest);
+
+        // verify
```

Comment:
Please replace the "// verify" parts with "// test" . We have everywhere (Sechub server, PDS, go client, plugins, wrapper applications, github-action) the combinations
```java
// prepare
// check preconditions
// execute
// test
```
I would prefer not to have a new version for "// test"

```suggestion
        // test
```